src/main/webapp/js/instructorStudentEdit.js

```diff
@@ -11,21 +11,51 @@ $(document).ready(function() {
 function readyInstructorStudentEditPage() {
     
     $(INSTRUCTOR_STUDENT_EDIT_FORM).on('submit', function(event) {
+        
+        var isEmailFieldChanged = $('#newstudentemail').val() !== $('#studentemail').val();
+        var isAnyEmailSentInThisCourse = $('#isAnyEmailSentForTheCourse').val();
+        
         if ($(INSTRUCTOR_STUDENT_EDIT_FORM).attr('editStatus') === 'mustDeleteResponses') {
+            isSubmitFormForEditingTeamName = false;
```

Comment:
Two questions here:
- Where is this variable defined?
- Where is this variable used?